src/VeresModel.cc

```diff
@@ -92,6 +92,43 @@ std::vector<double> VeresModel::gradient(std::vector<double> const& params) cons
     return gradChiSq;
 }
 
+std::tuple<double, std::vector<double>> VeresModel::computeFluxWithGradient(std::vector<double> const& params) const {
+
+    double xc = params[0];     // Centroid x
+    double yc = params[1];     // Centroid y
+    // double flux = params[2];   // Flux (in this case, always 1)
```

Comment:
should this be commented out?

Reply:
Yes. It made sense to me to keep the same function arguments as the other methods, so I just left it as a comment.